osu.Game/OsuGame.cs

```diff
@@ -577,6 +577,15 @@ protected override void LoadComplete()
 
             dependencies.CacheAs(idleTracker = new GameIdleTracker(6000));
 
+            GameIdleTracker sessionIdleTracker = new GameIdleTracker(300_000);
+            Add(sessionIdleTracker);
+
+            sessionIdleTracker.IsIdle.BindValueChanged(e =>
+            {
+                if (e.NewValue)
+                    Dependencies.Get<SessionStatics>().ResetValues();
+            });
```

Comment:
Kinda loath to see another tracker at the `OsuGame` level, because there's enough stuff there already, but not sure that it can be avoided... Maybe would put the reset logic in a subclass at least so that it's not out here.

Reply:
What do you mean by "subclass"? A nested class or a class that extends another?

Reply:
I was thinking about having a class inheriting `GameIdleTracker` but I'm not super sure of that idea myself. Probably need more feedback from others before proceeding with that proposal.

Reply:
I've tidied this up a bit but I think it's fine where it is.